examples/chart/teleport-cluster/charts/access_slack/Chart.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v2
+name: access-slack
+description: A Helm chart for Kubernetes
+
+# A chart can be either an 'application' or a 'library' chart.
+#
+# Application charts are a collection of templates that can be packaged into versioned archives
+# to be deployed.
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.1.0
```

Comment:
@webvictim It comes from `helm create` auto-generation because I put all the slack plugin stuff as a subchart.

I know the chart version should match the Teleport version but I'm afraid one can't link the `teleport-cluster` chart version to the version of its subchart and they have to change the version manually here